cmd/dump/dump.go

```diff
@@ -17,8 +18,9 @@ func main() {
 	pflag.DurationVarP(&duration, "since", "s", 10*time.Minute, "collection window, defaults to 10 minutes")
 	pflag.Parse()
 
+	isSRIOVLane := strings.Contains(os.Getenv("KUBEVIRT_PROVIDER"), "sriov")
 	// Hardcoding maxFails to 1 since the purpouse here is just to dump the state once
-	reporter := reporter.NewKubernetesReporter(os.Getenv("ARTIFACTS"), 1)
+	reporter := reporter.NewKubernetesReporter(os.Getenv("ARTIFACTS"), 1, isSRIOVLane)
```

Comment:
Can't we just try to collect it? If not there we will get a 404 and we just go on.

Reply:
We can, I don't have strong opinions nor arguments either for / against it.